Update for DDJ-200 4-deck mode (#237)
mixxxdj/manual: source/hardware/controllers/pioneer_ddj_200.rst

   :alt: Pioneer DDJ-200 (deck section)
   :figclass: pretty-figures

   Deck section (p. 8)

===  =================================================  ============================================================================================
No.  Control                                            Function
===  =================================================  ============================================================================================
1    :hwlabel:`JOG WHEEL` (top)                         Scratch (move play position)
1    :hwlabel:`JOG WHEEL` (outer)                       Pitch bend (nudge)
1    :hwlabel:`SHIFT` + `JOG WHEEL` (top)               Scratch (move play position) faster
1    :hwlabel:`SHIFT (left)` + `JOG WHEEL` (outer)      Move track selection in library
2    :hwlabel:`SHIFT` button                            Switch function of controls

> 3    :hwlabel:`PAD 1 - 8`                               Set (if empty) or play (if set) hot cue point / loop 1 - 8
> 3    :hwlabel:`SHIFT` + `PAD 1 - 8`                     Unset / delete hot cue 1 - 8

Reviewer: Same here:

```suggestion
3    Pad 1 - 8                                          Set (if empty) or play (if set) hot cue point / loop 1 - 8
3    :hwlabel:`SHIFT` + Pad 1 - 8                       Unset / delete hot cue 1 - 8
```

   :alt: Pioneer DDJ-200 (mixer section)
   :figclass: pretty-figures

   Mixer section (p. 10)

===  =================================================  ============================================================================================
No.  Control                                            Function
===  =================================================  ============================================================================================
1    :hwlabel:`MASTER CUE` button                       Toggle Master/Cue knob of headphones between left and right

> 1    :hwlabel:`SHIFT` + `MASTER CUE` button             Toggle between 2- and 4-deck mode

Reviewer: ```suggestion
1    :hwlabel:`SHIFT` + :hwlabel:`MASTER CUE` button    Toggle between 2- and 4-deck mode
```

2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs  Adjust high/mid/low-frequencies
3    :hwlabel:`CFX` knobs                               Turns on the selected effects

> 4    :hwlabel:`HEADPHONE CUE 1 / 2` button              Toggle headphone pre-fader listening of channel 1 / 2, in 4-deck mode LED idicate deck 3 and 4

Reviewer: ```suggestion
4    :hwlabel:`HEADPHONE CUE 1 / 2` button              Toggle headphone pre-fader listening of channel 1 / 2, in 4-deck mode LED indicate deck 3 and 4
```

What does this mean? It the LED lit if deck 3/4 is active? Then you should probably just write that instead.

> 4    :hwlabel:`SHIFT (left)` + `HEADPHONE CUE 1 / 2`    Load selected track to left or right selected deck
> 4    :hwlabel:`SHIFT (right)` + `HEADPHONE CUE 1 / 2`   In 4-deck mode: toggle between deck 1 / 3 and 2 / 4, LED indicate deck 3 and 4

Reviewer: ```suggestion
4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1 / 2`    Load selected track to left or right selected deck
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1 / 2`   In 4-deck mode: toggle between deck 1 / 3 and 2 / 4, LED indicate deck 3 and 4
```

> 4    :hwlabel:`HEADPHONE CUE 1 / 2` button              Toggle headphone pre-fader listening of channel 1 / 2, in 4-deck mode LED indicate deck 3 and 4
> 4    :hwlabel:`SHIFT (left)` + Cue :hwlabel:`1 / 2`     Load selected track to left or right selected deck
> 4    :hwlabel:`SHIFT (right)` + Cue :hwlabel:`1 / 2`    In 4-deck mode: toggle between deck 1 / 3 and 2 / 4, LED indicate deck 3 and 4

Reviewer: ```suggestion
4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1`   Load selected track to left deck.
4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 2`   Load selected track to right deck.
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1`  Toggle between left deck between 1 / 3 in 4-deck mode (if LED is lit, deck 3 is active).
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 2`  Toggle between right deck between 2 / 4 in 4-deck mode (if LED is lit, deck 4 is active).
```

(and realign the columns).

5    Channel faders                                     Adjust the output level for each channel
6    Transition FX Button                               Turns on :ref:`AutoDJ <djing-auto-dj>`

> 7    Cross fader                                        Fade between left and right deck

Reviewer: Please align the table columns properly, otherwise the manual will refuse to build. I can't post a suggestion though because GitHub refuses to let me.

Author: I aligned the mixer table columns.

===  =====================================================  ============================================================================================
No.  Control                                                Function
===  =====================================================  ============================================================================================
1    :hwlabel:`MASTER` button                               Toggle Master/Cue knob of headphones between left and right
1    :hwlabel:`SHIFT` + :hwlabel:`MASTER` button            Toggle between 2- and 4-deck mode
2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs      Adjust high/mid/low-frequencies
3    :hwlabel:`CFX` knobs                                   Turns on the selected effects
4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1`   Load selected track to left deck.
4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 2`   Load selected track to right deck.

> 4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1`  Toggle between left deck between 1 / 3 in 4-deck mode (if LED is lit, deck 3 is active).
> 4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 2`  Toggle between right deck between 2 / 4 in 4-deck mode (if LED is lit, deck 4 is active).

Reviewer: Is this correct? Then I think this would be a good clarification.

```suggestion
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1`  Toggle between left deck between 1 / 3 in 4-deck mode (if LED is lit, deck 3 is active). If 4-deck mode is disabled, this behaves the same as :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1`.
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 2`  Toggle between right deck between 2 / 4 in 4-deck mode (if LED is lit, deck 4 is active). If 4-deck mode is disabled, this behaves the same as :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 2`.
```
